strapi/strapi
# Fix bulk delete when entity has empty components

### What does it do?

Fix https://github.com/strapi/strapi/issues/14384

### Why is it needed?

Deleting an entity with empty components was throwing an error.
`deleteComponents` function expects one of the two:
- An entity with loaded components.
- An entity with an id.

When components were empty, the first case was not met, so the function was expecting the entity with an id, which we were not passing. So 💥 

### How to test it?

Delete or bulk delete an entity and you should not see any error in the console.

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/14384
